include/klee/ExecutionState.h

```diff
@@ -145,6 +145,9 @@ class ExecutionState {
   //@brief Symbolic error information
   SymbolicError *symbolicError;
 
+  //@breif Used to generate return variable names for math function calls
+  int mathVarCount;
```

Comment:
@Himeshi I recommend moving this field to `ErrorState` so that we have the math and math error variables are created in the same place.